src/test/java/com/epam/tc/hw3/tests/PagesUITest.java

```diff
@@ -0,0 +1,39 @@
+package com.epam.tc.hw3.tests;
+
+import com.epam.tc.hw3.utils.WebDriverProvider;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+public class PagesUITest {
+
```

Comment:
since it's a base class it's better to make it abstract

Reply:
Done